### PR TITLE
test: fix invalid otp response

### PR DESCRIPTION
**Pull request by worldleelavivat-okta**
## Description:

- Update Invalid OTP response to correct response

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-407272](https://oktainc.atlassian.net/browse/OKTA-407272)

### CODE REVIEW

File: playground/mocks/config/responseConfig.js [hunk 1]
```diff
@@ -756,5 +756,5 @@ const googleAuthenticatorVerify = {
 };
 
 module.exports = {
-  mocks: idx,
+  mocks: windowAuthnLoopbackPollingFail,
```
Comment (pradeepdewda-okta) [inline playground/mocks/config/responseConfig.js:759]: change it back to idx

**Review by pradeepdewda-okta — APPROVED** [review]

**Comment by pradeepdewda-okta** [comment]
@worldleelavivat-okta remaining flows related to Cred, seems fine right ?

File: playground/mocks/config/responseConfig.js (hunk 1 above)
Comment (worldleelavivat-okta) [reply playground/mocks/config/responseConfig.js:759]: will change it back

**Comment by worldleelavivat-okta** [comment]
@pradeepdewda-okta everything else seem fine. I checked other error within authenticator flow (phone/email) and all of them are up to date as I am aware.

**Pull request closed without merging**